### DISCUSSION TITLE
Where can i import props of TextProps

### DISCUSSION THREAD

**Discussion by Babailan** [radix-ui/themes#277] [General]
https://github.com/radix-ui/themes/blob/main/packages/radix-ui-themes/src/components/text.tsx#L22
Is there a way to import this TextProps type

**Comment by vladmoroz** [comment]
The component prop types are not exported. However, you can use React’s built-in type helpers to obtain props of any component:

```ts
import * as React from 'react'
import { Text } from '@radix-ui/themes';
type TextProps = React.ComponentPropsWithoutRef<typeof Text>
```

**Reply by Babailan** [reply]
Thank you for feedback
